git-github-introduction.md

```diff
@@ -0,0 +1,383 @@
+(Version Control System).  A VCS helps manage changes made to files.  Users can run Git on their local
+computers and upload changes to remote repositories and files hosted on GitHub.  Contributors can edit files
+locally, upload their changes to GitHub, and then initiate a pull request. The following document
+covers the basics of using Git and submitting a pull request to one of our project's GitHub repository.
+
+
+
+
+
+## Git Installation
+[git-scm.com](https://git-scm.com/) provides platform-specific installers for
+Mac and Windows users.  Installation instructions are provided in the links
+below for each supported platform.
+
+[Mac](https://git-scm.com/download/mac)
+
+[Windows](https://git-scm.com/download/win)
+
+[Linux](http://git-scm.com/download/linux)
+
+
+## Configure Git User Information
+
+After installing Git, the next step is for the user to configure his/her author information (username and email address) so that Git will correctly log the author of a commit. 
+
+
+To configure Git author information, open the Terminal and type **git config --global user.name "user's name"**, which will allow you to configure Git on your local machine. Each repository you create will use the same user (author) configuration if you use the global flag (as in the provided example). 
+
+```
+$ git config --global user.name "John Smith"
+
+$ git config --global user.email "john.smith@example.com"
+```
+
+You can check your user information by typing the following command:
+
+```
+$ git config --list
+```
+
+## Configure Git to sign commits
+
+If [GPG](https://gnupg.org/) is installed on your system, you can configure Git
+to sign your commits.
+
+```
+$ git config --global user.signingkey <your GPG key ID>
+```
+
+Instructions on how to tell Git about your GPG key can be found [here](https://help.github.com/articles/telling-git-about-your-gpg-key/).
+
+There is also a convenient Git command to automatically sign all of of your commits:
+Note: The command below requires Git version >= 2.0.0.
+
+```
+$ git config commit.gpgsign true
+```
+
+
+## Configure GitHub User Account
+Before a pull request can be submitted to a [SeattleTestbed](https://github.com/SeattleTestbed)
+repository, a user account (free) must be created on GitHub.  This user account is also needed to create
+new repositories and fork ones that already exist on GitHub.
+
+
+A GitHub user account can be created at https://github.com/join
+
+Once users create their accounts, user-specific home pages are created. For
+example: ''https://github.com/john-smith''.  A user-specific home page can
+contain repositories that the user has created or forked, contribution
+activity, and organizations he/she has joined.
+
+## Fork a Repository
+
+Forking refers to making a copy of a repository and saving it to your user home page. General instructions on forking repositories and managing pull requests are available [here](https://help.github.com/articles/fork-a-repo).
```

Comment:
Maybe clarify that *forking* is a *GitHub* and not a *git* concept (plus replace the `here`-link)? E.g.

*"Forking refers to saving a copy of a repository to your *GitHub* user home page. *GitHub* provides general instructions on [creating and managing forks](https://help.github.com/articles/fork-a-repo/) and [about pull requests](https://help.github.com/articles/about-pull-requests/).*